test/run_test_bort.sh

```diff
@@ -5,12 +5,54 @@
 #
 # Ed Hartnett 3/12/23
 
+# Check adn30().
+./test_bort_4 adn30 1
+[ $? != 1 ] &&  exit 1
+./test_bort_4 adn30 2
+[ $? != 1 ] &&  exit 1
+
+# Check bort().
 ./test_bort_4 bort 1
 [ $? != 1 ] &&  exit 1
 
-./test_bort_4 bort 2
+# Check bort2().
+./test_bort_4 bort2 1
 [ $? != 1 ] && exit 1
 
+# Check bvers().
+./test_bort_4 bvers 1
+[ $? != 1 ] &&  exit 1
+
+# Check cmpmsg().
+./test_bort_4 cmpmsg 1
+[ $? != 1 ] &&  exit 1
+
+# Check wtstat().
+./test_bort_4 wtstat 1
+[ $? != 1 ] &&  exit 1
+./test_bort_4 wtstat 2
+[ $? != 1 ] &&  exit 1
+./test_bort_4 wtstat 3
+[ $? != 1 ] &&  exit 1
+./test_bort_4 wtstat 4
+[ $? != 1 ] &&  exit 1
+
+# Check writdx().
+./test_bort_4 writdx 1
+[ $? != 1 ] &&  exit 1
+
+# Check writlc().
+./test_bort_4 writlc 1
+[ $? != 1 ] &&  exit 1
+
+# Check writsa().
+./test_bort_4 writsa 1
+[ $? != 1 ] &&  exit 1
+
+# Check writsb().
+./test_bort_4 writsb 1
+[ $? != 1 ] &&  exit 1
+
 # If we made it here, all error codes were correctly returned, and the
```

Comment:
What about tests for test_bort_8 and test_bort_d?

Reply:
Still coming. But since bort only accepts char strings anyway, this does not seem urgent. However we'll get there by the end of the week...